factory/wsFactory.go

```diff
@@ -24,6 +24,7 @@ func CreateWSHandler(apiType string, publisher process.Publisher, marshaller mar
 	hub, ok := publisher.(dispatcher.Hub)
 	if !ok {
 		log.Warn("unable to cast publisher to hub")
```

Comment:
I would not return error and also log as warn. Reason? This can lead to double error logging.

Reply:
removed the log warn